### PR TITLE
[bug] Accessibility issues addressed (closes 1256, 1289, 1296)

### PR DESCRIPTION
**Pull request by i336616**
https://jira.itc.sap.com/browse/EXPOSUREAPP-1256
https://jira.itc.sap.com/browse/EXPOSUREAPP-1289
https://jira.itc.sap.com/browse/EXPOSUREAPP-1296

- Added accessibility to text content
- Updated accessibility for HomeRiskLevelCellConfigurator
- Added accessibility to HomeActivateCellConfigurator
- Added accessibility to HomeTextResultCollectionViewCell
- Added image accessibility to ExposureSubmissionIntroViewController (label = "" for now)
- Adde accessibility to image on Exposure Settings screen
- Add trait header to Infektionsrisiko
- Updated accessibility for HomeTestResultCollectionViewCell

Added UITest for generating screenshots for the AppStore

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/HomeTestResultCollectionViewCell.swift [hunk 1]
```diff
@@ -85,4 +87,14 @@ class HomeTestResultCollectionViewCell: HomeCardCollectionViewCell {
...
+
+	func setupAccessibility() {
+		titleLabel.isAccessibilityElement = true
+		subtitleLabel.isAccessibilityElement = true
+		descriptionLabel.isAccessibilityElement = true
+		illustrationView.isAccessibilityElement = true
+		isAccessibilityElement = false
+
+		titleLabel.accessibilityTraits = .header
+	}
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/HomeTestResultCollectionViewCell.swift:99]: Could you please align with this one: https://github.com/corona-warn-app/cwa-app-ios/blob/db44967876567ca966a0431d1dc4cbe9230da66e/src/xcode/ENA/ENA/Source/Views/Home%20Screen/Cells/HomeTestResultCollectionViewCell.swift#L51

**Review by eisvogel — CHANGES_REQUESTED** [review]
mini change, please

File: src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/HomeTestResultCollectionViewCell.swift (hunk 1 above)
Comment (i336616) [reply src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/HomeTestResultCollectionViewCell.swift:99]: Merged that in.

**Comment by EisvogelSAP** [comment]
1289 was already fixed with https://jira.itc.sap.com/browse/EXPOSUREAPP-1391
in https://github.com/corona-warn-app/cwa-app-ios/pull/470

Please align.

**Pull request closed without merging**